.github/CODEOWNERS

```diff
@@ -11,6 +11,7 @@
 /plugins/cost-insights                      @backstage/silver-lining
 /plugins/cloudbuild                         @trivago/ebarrios
 /plugins/search                             @backstage/techdocs-core
+/plugins/search-*                           @backstage/techdocs-core
```

Comment:
Do you want to also add something like

```text
/packages/app/src/components/search/*       @backstage/techdocs-core
```

Since sample code is being placed into the example app?